rclcpp/include/rclcpp/subscription_options.hpp

```diff
@@ -66,7 +66,7 @@ struct SubscriptionOptionsBase
     // Topic to which topic statistics get published when enabled. Defaults to /statistics.
     std::string publish_topic = "/statistics";
 
-    // Topic statistics publication period in ms. Defaults to one minute.
+    // Topic statistics publication period in ms. Defaults to one second.
```

Comment:
We should document there that only values `>= 0` are allowed. And/or we should update the functions that throw to say they can throw in this situation.

Reply:
Added doc